ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PackageUploadLedgerEntry.scala

```diff
@@ -5,27 +5,25 @@ package com.digitalasset.platform.sandbox.stores.ledger
 
 import com.daml.ledger.participant.state.v1.ParticipantId
 
-sealed abstract class PackageUploadEntry(
-    )
-    extends Product
-    with Serializable {
+//TODO BH consider removing duplication of this class
+sealed abstract class PackageUploadLedgerEntry() extends Product with Serializable {
   val submissionId: String
   val participantId: ParticipantId
   def value: String
 }
 
-object PackageUploadEntry {
+object PackageUploadLedgerEntry {
```

Comment:
The entries are missing record time.

Reply:
now includes recordTime and wired in to store this to DB